circuits/transfer/src/send_to_contract_obfuscated.rs

```diff
@@ -205,7 +205,7 @@ impl SendToContractObfuscatedCircuit {
 }
 
 #[allow(clippy::option_map_unit_fn)]
-#[code_hasher::hash(name = "CIRCUIT_ID", version = "0.1.0")]
+#[code_hasher::hash(name = "CIRCUIT_ID", version = "0.2.0")]
```

Comment:
Can you open an issue that explain why you had to bump the version? so to avoid not necessary bump in the future

Reply:
The hashing of the circuits will likely change with the new plonk circuit compression scheme #949. I'll keep this issue in mind